### PR TITLE
[test]Add schema change for regression test

### PR DESCRIPTION
**Pull request by zy-kkk**
# Proposed changes

Issue Number: close #xxx

## Problem Summary:

Describe the overview of changes.

## Checklist(Required)

1. Does it affect the original behavior: (Yes/No/I Don't know)
2. Has unit tests been added: (Yes/No/No Need)
3. Has document been added or modified: (Yes/No/No Need)
4. Does it need to update dependencies: (Yes/No)
5. Are there any changes that cannot be rolled back: (Yes/No)

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: regression-test/suites/schema_change/test_alter_table_column.groovy [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+    sql """
+            CREATE TABLE IF NOT EXISTS ${tbName1} (
+                k1 INT,
+                value1 INT
+            )
+            DUPLICATE KEY (k1)
+            DISTRIBUTED BY HASH(k1) BUCKETS 5 properties("replication_num" = "1");
+        """
+    String res = "null"
+    sql "ALTER TABLE ${tbName1} ADD COLUMN k2 INT KEY AFTER k1"
```
Comment (cambyzju) [inline regression-test/suites/schema_change/test_alter_table_column.groovy:30]: It is better to test alter table with some data.

Comment (zy-kkk) [reply regression-test/suites/schema_change/test_alter_table_column.groovy:30]: I have added the test data, thank you for your correction

File: regression-test/suites/schema_change/test_alter_table_column.groovy [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+    res = "null"
+    while (!res.contains("FINISHED")){
+        res = sql "SHOW ALTER TABLE COLUMN WHERE TableName='${tbName2}' ORDER BY CreateTime DESC LIMIT 1;"
+        Thread.sleep(1000)
+    }
+    res = "null"
+    sql "ALTER TABLE ${tbName2} ADD COLUMN value2 INT SUM AFTER value1"
+    while (!res.contains("FINISHED")){
+        res = sql "SHOW ALTER TABLE COLUMN WHERE TableName='${tbName2}' ORDER BY CreateTime DESC LIMIT 1;"
+        Thread.sleep(1000)
```
Comment (BiteTheDDDDt) [inline regression-test/suites/schema_change/test_alter_table_column.groovy:92]: What happens when the `alter table` task fails?

Comment (zy-kkk) [reply regression-test/suites/schema_change/test_alter_table_column.groovy:92]: I think I should improve the loop and add the recognition of failed states

**Review by yiguolei — APPROVED** [review]
LGTM

**Pull request merged**